pyproject.toml

```diff
@@ -79,10 +89,35 @@ dev-dependencies = [
     "types-pyyaml>=6.0.12.20250516",
     "requests>=2.32.4",
     "urllib3>=2.5.0,<3.0.0",
+    "httpx<0.26.0",
+    "referencing>=0.36.2",
     "cffconvert",
 ]
 
+conflicts = [[{ extra = "torch" }, { extra = "torch-cu126" }]]
+prerelease = "allow"
+
+[[tool.uv.index]]
+name = "pytorch-cpu"
+url = "https://download.pytorch.org/whl/cpu"
+explicit = true
+
+[[tool.uv.index]]
+name = "pytorch-cu126"
+url = "https://download.pytorch.org/whl/cu126"
+explicit = true
+
 [tool.uv.sources]
+torch = [
+  { index = "pytorch-cpu", extra = "torch" },
+  { index = "pytorch-cu126", extra = "torch-cu126" },
+]
+torchvision = [
+  { index = "pytorch-cpu", extra = "torch" },
+  { index = "pytorch-cu126", extra = "torch-cu126" },
+]
+pytorch-triton = [{ index = "pytorch-cu126", extra = "torch-cu126" }]
+torchgeo = { git = "https://github.com/microsoft/torchgeo", rev = "95dba5c43a2828c04a6b0a316d1996c5d1c6b9c5" }
```

Comment:
when can we pin this to a release version?

Reply:
This is only needed for testing. shall we move it to the dev dependencies instead?

Reply:
Could also move it to a torchgeo extras

Reply:
I think the current `torch`/`torch-cu126` extras are sufficiently granular. I wouldn't move it to dev since that will make alignment with these extra complicated. Having tests that need `torch`/`torchgeo` do a conditional check is sufficient IMO.

Reply:
@isaaccorley to be clearer, I meant when can we pin torchgeo to a stable version range here instead of a git commit?

Reply:
Ahh I see what you mean, we could pin to torchgeo 0.7.1 but FTW won't be in main until 0.8.0 which should happen soon after the TorchGeo transition next week.

Reply:
Right now I'm actually not even running the tests that require torchgeo because they are slow. Shall we remove the torchgeo tests and torchgeo dependency entirely from the PR and follow up after with the other torchgeo PR possibly in https://github.com/stac-extensions/mlm/pull/102

Reply:
I removed torchgeo tests and dependency for now. We have tests with just a plain conv model as a replacement for the unet that tests the export/package end-to-end. Will add in a follow-up PR once a new release is out.